src/gui/update.rs

```diff
@@ -509,6 +538,37 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
             ajour.header_state.previous_sort_direction = Some(sort_direction);
             ajour.header_state.previous_sort_key = Some(sort_key);
         }
+        Message::ReleaseChannelSelected(release_channel) => {
+            log::debug!("Message::ReleaseChannelSelected({:?})", release_channel);
+
+            if let Some(expanded_addon) = ajour.expanded_addon.clone() {
```

Comment:
Unnecessary clone, use `&ajour.expanded_addon` instead. We don't need an owned version of this.

Reply:
If a type is not `Copy`, it will try to move that value / "take ownership" and the borrow checker will obviously complain. Always default to using the borrowed value and instead have the borrow checker tell you when you need an owned version. That should help prevent adding an unneeded clone just to get the borrow checker to stop complaining.

Reply:
What editor do you use? Are you using Rust Analyzer? I find the `inlay hints` it provides extremely helpful in cases like this. For example here's what VS Code shows me before and after. Notice the `inlay hint` of `Addon` before and `&Addon` after. It makes it really easy to see that we were using an owned value and now a borrowed value.

![image](https://user-images.githubusercontent.com/10239377/94319945-9621aa80-ff40-11ea-94d2-c4888b1f6951.png)

![image](https://user-images.githubusercontent.com/10239377/94319956-a20d6c80-ff40-11ea-9c7d-2e0b08aca8ff.png)

Reply:
Using Vim, but with Rust Analyzer. Will see if I can add the inline hits like that as well.

<img width="778" alt="Screenshot 2020-09-26 at 07 43 01" src="https://user-images.githubusercontent.com/2248455/94331344-f873bd00-ffcb-11ea-9024-4e31e53d497a.png">